app/controllers/news_controller.rb

```diff
@@ -13,9 +11,9 @@ def index
     if params[:section].present?
       set_section
       set_menu_nodes
-      @articles = news_articles section: @section
+      @articles = preload_news_articles(@section.news_articles)
```

Comment:
Do we need to pass `@section`? If the helper is included, it would have access to it by default.